tap/authregister_backend.go

```diff
@@ -2,11 +2,18 @@
 proxy */
 package tap
 
+import "github.com/TykTechnologies/tyk/apidef"
+
 // AuthRegisterBackend is an interface to provide storage for profiles loaded into TAP
 type AuthRegisterBackend interface {
 	Init(interface{})
-	SetKey(key string, orgId string, val interface{}) error
+	SetKey(key string, orgId string, val DBObject) error
 	GetKey(key string, orgId string, val interface{}) error
```

Comment:
Why GetKey and GetAll still an interface?

Reply:
It has implementations for Redis and InMemory, so I didn't touch to the interface.

Reply:
https://github.com/TykTechnologies/tyk-analytics/blob/371ad360e6b22ceb23829aa417ded6d4ff35bda8/dashboard/api_internal_identity_broker.go#L192